pkg/addons/data/Service/default/router.yaml

```diff
@@ -20,3 +20,4 @@ spec:
     port: 1936
```

Comment:
1936 should probably not be on the external service - separate service + update prometheus?

Reply:
So make it 2 services `router-external` and `router` ?

Reply:
router and router-stats?